<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LowLevelTest</class>
 <widget class="QDialog" name="LowLevelTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>492</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove any ICs from socket, turn all DIP switches off, remove wire links.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPowerOn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+5V On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAllOn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>All on</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAllOff">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>All off</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>86</y>
     <width>201</width>
     <height>332</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>24</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn24">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">24</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn23">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">23</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn22">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">22</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn21">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">21</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn20">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">20</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn19">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">19</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btn18">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">18</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="btn8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">8</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btn17">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">17</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="btn9">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">9</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="btn16">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">16</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="btn10">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">10</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="btn15">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">15</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="btn11">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">11</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="btn14">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">14</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="btn12">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">12</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="btn13">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string notr="true">13</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LowLevelTest</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LowLevelTest</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
